### DISCUSSION TITLE
AttributeError: 'Steps' object has no attribute 'get_artifact'

### DISCUSSION THREAD

**Discussion by remicres** [argoproj-labs/hera#1282] [Q&A] [answered]
Hi,

I am surprised to see that `Steps` has no `get_artifact()`. Is that a bug or intended? 
It has `get_parameter()` but no `get_artifact()`, very strange

**Comment by elliotgunton** [comment, marked as answer]
This seems like a small oversight when adding the `get_parameter` in https://github.com/argoproj-labs/hera/issues/816. Will turn this into an issue, thanks!

**Comment by elliotgunton** [comment]
Please see #1283
